src/App.js

```diff
@@ -53,20 +89,29 @@ function App() {
       </Navbar>
       <Container>
         <Row>
-          <h1>this is a thing now</h1>
+          {/* render grocery item */}
+          {mockFetchItems.map(item => (
```

Comment:
![image](https://user-images.githubusercontent.com/7408875/66702623-5ffe1180-ed55-11e9-8708-7399e708e443.png)

Forgot to mention with rendering arrays of components using map, each component needs a `key` prop. It can be anything unique, and it supposed to be there to help React know which elements in an array of items have changed, between rerender (so react only has to rerender the changed items.)

```
<GroceryItem 
  itemTitle={item.title} 
  itemDesc={item.desc}
  imgUrl={item.img} 
  key={item.title}
/>
```

Reply:
Done!